dev/TabView/TabViewListView.cpp

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include "ResourceAccessor.h"
+#include "Utils.h"
+#include "TabViewListView.h"
+#include "TabViewItem.h"
+#include "TabView.h"
+#include "SharedHelpers.h"
+
+CppWinRTActivatableClassWithBasicFactory(TabViewListView);
+
+// IItemsControlOverrides
+
+winrt::DependencyObject TabViewListView::GetContainerForItemOverride()
+{
+    return winrt::make<TabViewItem>();
+}
+
+bool TabViewListView::IsItemItsOwnContainerOverride(winrt::IInspectable const& args)
+{
+    bool isItemItsOwnContainer = false;
+    if (args)
+    {
+        auto item = args.try_as<winrt::TabViewItem>();
```

Comment:
try_as works against null so this can be combine with the above if:
```
if (auto item = args.try_as<TabViewItem>())
{
```

Reply:
oh neat, didn't realize